source/_integrations/unifiprotect.markdown

```diff
@@ -0,0 +1,84 @@
+ha_category:
+  - Hub
+  - Camera
+ha_release: 2022.2
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_quality_scale: platinum
+ha_codeowners:
+  - '@briis'
+  - '@AngellusMortis'
+  - '@bdraco'
+ha_domain: unifiprotect
+ha_platforms:
+  - camera
+---
+
+The UniFi Protect integration, adds support for retrieving Camera feeds and Sensor data from an [UniFi Protect application](https://ui.com/camera-security) by [Ubiquiti Networks, inc.](https://www.ui.com/) that is running on an UniFi OS Console.
+
+## Prerequisites
+
+### Hardware Support
+
+This Integration supports all UniFi OS Consoles that can run UniFi Protect. Currently this includes:
+
+* UniFi Protect Network Video Recorder (**UNVR**)
+* UniFi Protect Network Video Recorder Pro (**UNVRPRO**)
+* UniFi Dream Machine Pro (**UDMP**)
+* UniFi Cloud Key Gen2 Plus (**CKGP**) firmware version v2.0.24+
+
+CKGP with Firmware v1.x **do NOT run UniFiOS**, you must upgrade to firmware v2.0.24 or newer.
+
+### Software Support
+
+The absolute **minimal** software version is `1.20.0` for UniFi Protect. If you have an older version, you will get errors trying to set up the integration. However, the general advice is the latest 2 minor versions of UniFi Protect and hardware supported by those are supported. Since UniFi Protect has its own release cycle, you should only upgrade UniFi Protect _after_ the next Home Assistant release comes out to ensure the new version is fully supported.
+
+Example: as of `2022.2.0` of Home Assistant, UniFi Protect `1.21.0-beta.1` is the newest UniFi Protect version. So the recommended versions of UniFi Protect to run for a `2022.2.0` version of Home Assistant are `1.20.0`, `1.20.1` or `1.21.0-beta.1`.
+
+### Local User
+
+You will need a local user created in your UniFi OS Console to log in with. Ubiquiti Cloud Users will **not** work.
+
+1. Login to your *Local Portal* on your UniFiOS device, and click on *Users*
+1. In the upper right corner, click on *Add User*
+1. Click *Add Admin*, and fill out the form. Specific Fields to pay attention to:
+    * Role: Must be *Limited Admin*
+    * Account Type: *Local Access Only*
+    * CONTROLLER PERMISSIONS - Under UniFi Protect, select Administrators. **NOTE**: Other roles may work, but only the default Administrators rule is fully tested.
+1. Click *Add* in at the bottom Right.
+
+**HINT**: A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try restart the UDMP.
+
+![ADMIN_UNIFIOS](/images/integrations/unifiprotect/unifi_os_admin.png)
+
+### Camera Streams
+
+The Integration uses the RTSP(S) Streams as the Live Feed source, so this needs to be enabled on each camera to ensure you can stream your camera in Home Assistant. This may already be enabled by default, but it is recommended to just check that this is done. To check and enable the the feature
```

Comment:
```suggestion
The Integration uses the RTSP(S) Streams as the Live Feed source, so this needs to be enabled on each camera to ensure you can stream your camera in Home Assistant. This may already be enabled by default, but it is recommended to just check that this is done. To check and enable the feature:
```